cmd/minikube/cmd/start.go

```diff
@@ -1208,6 +1212,18 @@ func validateRegistryMirror() {
 	}
 }
 
+// This function validates if the --image-repository
+// args match the format of registry.cn-hangzhou.aliyuncs.com/google_containers
+func validateImageRepository(imagRepo string) {
+	URL, err := url.Parse(imagRepo)
+	if err != nil {
+		klog.Errorln("Error Parsing URL: ", err)
+	}
+	if URL.Scheme != "" || strings.HasSuffix(URL.Path, "/") {
+		exit.Message(reason.Usage, "Sorry, the url provided with the --image-repository flag is invalid: {{.url}}", out.V{"url": imagRepo})
```

Comment:
instead of exiting, how about fixing it (remove the trailing '/")  for them and provide a info
something like
```
out.Infof("The --image-repository flag u provided ended with a trailing / that could cause conflict in kuberentes, removed automatically"
```

Reply:
thanks for your reply. I will change it.